website/docs/docs/deploy/job-scheduler.md

```diff
@@ -82,6 +83,23 @@ The scheduler prevents queue clog by canceling runs that aren't needed, ensuring
 
 To prevent over-scheduling, users will need to take action by either refactoring the job so it runs faster or modifying its [schedule](/docs/deploy/deploy-jobs#schedule-days).
 
+## Deactivation of abandoned jobs
+
+The scheduler will consider a job abandoned if it meets these criteria: 
+- It's had 10 or more consecutive run errors
+- It's been 10 or more days since its last successful run
+- No code changes to the branch since the first failure
+
+Abandoned jobs, typically, are not used by anyone and with no one monitoring them becomes a performance drain and load balancing issue for the scheduler. To mitigate this situation, the scheduler will deactivate a job if it meets the defintion for abandoned and the job will no longer be enqueued to run. 
+
+Before deactivation, dbt will send a notification to warn you about it. Notifications are sent when a job has had 5 or more sequential run errors and it has been over 5 days since its last successful run with no code changes since the first failure. 
```

Comment:
are there more details regarding notifications that i can add here (eg email, slack, etc) ?

Reply:
Would be good to clarify who the email is sent to. The account owner? The person who made the job?